src/analysisd/decoders/winevtchannel.c

```diff
@@ -247,32 +248,7 @@ int DecodeWinevt(Eventinfo *lf){
                                         filtered_string = replace_win_format(child_attr[p]->content);
                                         *child_attr[p]->values[l] = tolower(*child_attr[p]->values[l]);
 
-                                        // Save category ID
-                                        if (!strcmp(child_attr[p]->values[l], "categoryId")){
```

Comment:
`categoryId`, `subcategoryId` and `auditPolicyChangesId` are no longer used, but they are still declared and freed when calling the `cleanup`:
https://github.com/wazuh/wazuh/blob/0ca25023c65bfa1838397f893b6b2963e9c28e0f/src/analysisd/decoders/winevtchannel.c#L80
https://github.com/wazuh/wazuh/blob/0ca25023c65bfa1838397f893b6b2963e9c28e0f/src/analysisd/decoders/winevtchannel.c#L475